zdm/survey.py

```diff
@@ -971,7 +1026,7 @@ def make_widths(s:Survey,state):
         # this is done for tests, for complex surveys such as CHIME,
         # or for estimating the properties of a single FRB
         weights.append(1.)
-        widths.append(np.exp(slogmean))
```

Comment:
I'm noting the question about "should this be wlogmean". Actually, yes, it "should", although usually that won't matter at all. Can you please change it and remove this #NOTE ?